src/app/components/publiclists/challengelist/challengelist.component.ts

```diff
@@ -302,6 +326,9 @@ export class ChallengelistComponent implements OnInit {
           SELF.pastChallenges = data['results'];
         }
         SELF.filteredChallenges = SELF.upcomingChallenges.concat(SELF.ongoingChallenges, SELF.pastChallenges);
+        SELF.filteredOngoingChallenges = SELF.ongoingChallenges;
```

Comment:
Why not use the existing variables itself, why there is need of new variable ?

Reply:
As i am displaying the count of `ongoing`, `past` and `upcoming` challenges separately and also displaying those challenges in separate tabs so i need to store them separately and so i need to use these new variables.Will add proper comment so that it would be easy to understand it's purpose.